scripts/patch_jpegdec.py

```
@@ -1,68 +1,85 @@
-Fix:
-  Redirect pMCU to sMCUs[0] when MCU_SKIP is active.  Writes to sMCUs[1..63]
-  are harmless: for JPEG_SCALE_EIGHTH only sMCUs[0] is read for output, and
-  the DC write at sMCUs[0] is already guarded by the existing `if (iMCU >= 0)`
-  check.
-
-Applied idempotently — safe to run on every build.
+Patches live in `scripts/jpegdec_patches/` as one-commit-per-fix files
+(see the file headers for context). Applied in lexical order.
 """
 
 Import("env")
```

Ruff (0.15.13)

[error] Undefined name `Import`

(F821)